pkg/machines/machines.less

```diff
@@ -149,3 +149,12 @@
 .dropdown-buttons-container {
     display: inline-flex !important;
 }
+
+/* Make the max-height based on the screen hight - offset from the top */
+#vendor-select .dropdown-menu {
```

Comment:
Seems like styling specific just for the CreateVmDialog. Can we move it to createVmDialog.css ?